style.css

```diff
@@ -28,11 +28,14 @@ nav {
 }
 i { margin-right: 5%;
 }
+.icon { transform: scale(1.3); }
 
 .iconu {
   color:red;
+  margin-right: 3%;
+  transform: scale(2);
 }
-.icon { transform: scale(1.3); }
+
 
 .notification {
   display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Hey .. Thank you very much for your review. We have learned a lot from your constructive way of pointing out our errors. This is just to inform you that we did commit messages plenty of time but the commit messages are not appearing in GitHub for some reason that we don't know. This is the first time we have worked with branches and probably we made mistakes while pushing branches to Github. We will check all the errors and after fixing it we will request for a recheck. Thank you very much for your quick feedback once again.

Reply:
Hei,
I shared our project link with my standup group and all of them can see the thumbnails picture for the recommended video just fine. I have also checked it by myself with another laptop and there was no that kind of issue that you mentioned.